pyk/src/pyk/kast/manip.py

```diff
@@ -749,7 +769,13 @@ def build_rule(
       - `rule`: A `KRule` with variable naming conventions applied so that it should be parseable by K frontend.
       - `var_map`: The variable renamings that happened to make the claim parseable by K frontend (which can be undone to recover original variables).
     """
-    init_constraints = list(init_constraints)
+    init_constraints = [normalize_ml_pred(c) for c in init_constraints]
+    final_constraints = [normalize_ml_pred(c) for c in final_constraints]
+    print('Initial constraints:')
+    print(*init_constraints, sep='\n')
+    print('Final constraints:')
+    print(*final_constraints, sep='\n')
+    print('----------------------------')
```

Comment:
No printing please, except for top level CLI commands. Either remove this or turn it into log messages at the appropriate log level (probably `debug`).

Reply:
Probably just remove this.

Reply:
Yes yes, this was not supposed to be pushed.